src/index.js

```diff
@@ -0,0 +1,142 @@
+let currentTemperature = 0;
+
+const tempValue = document.getElementById('tempValue');
+
+const temperatureDisplay = () => {
+    tempValue.textContent = currentTemperature;
+    tempColor();
+};
+
+const addDegree = () => {
+    currentTemperature += 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const subtractDegree = () => {
+    currentTemperature -= 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const tempColor = () => {
+    const BackgroundColor = document.getElementById('currentTempButton');
+
+    const elementFontColor = document.getElementById('tempValue');
+    let currentColor;
+    
+    const elementLandscape = document.getElementById('landscape');
+    let currentLandscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+
+    if (currentTemperature <= 49) {
+        currentColor = '#30849D';
+    } else if (currentTemperature >= 50 && currentTemperature <= 59 ) {
+        currentColor = '#50A63F';
+    } else if (currentTemperature >= 60 && currentTemperature <= 69 ) {
+        currentColor = '#FEE086';
+        currentLandscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (currentTemperature >= 70 && currentTemperature <= 79 ) {
+        currentColor = '#F8B5A2';
+        currentLandscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (currentTemperature >= 80) {
+        currentColor = '#E06767';
+        currentLandscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    }
+    BackgroundColor.style.backgroundColor = currentColor;
+    elementFontColor.style.color = currentColor;
+    elementLandscape.textContent = currentLandscape;
+};
+
+const citySky = (event) => {
+    const selectedOption = event.target.value;
+    const elementSky = document.getElementById('sky');
+    let currentCitySky = '';
+
+    if (selectedOption === 'sunny') {
+        currentCitySky = '☀️ ☁️ ☀️ ☀️ ☁️ ☀️ ☁️ ☀️ ☀️ ☁️ ☀️';
+    } else if (selectedOption === 'cloudy') {
+        currentCitySky = '☁️☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️☁️';
+    } else if (selectedOption === 'rainy') {
+        currentCitySky = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    } else if (selectedOption === 'snowy') {
+        currentCitySky = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    }
+    elementSky.textContent = currentCitySky;
+};
+
+const cityNameInput = document.getElementById('cityNameInput');
+const cityHeaderElement = document.getElementById('defaultHeaderCityName');
```

Comment:
Prefer that you put all these `getElementById` calls in one place (at the top of the file is fine) to make this file more organized, which ultimately makes a codebase more maintainable. Currently they're written throughout the file and one has to go looking for them or they could be unintentionally overlooked.